src/coach/wod_service.py

```diff
@@ -0,0 +1,24 @@
+import random
+from src.coach.models_db import WODModel
+from src.coach.database import db_session
+
+
+def create_wod_for_user(user_id: str, date: str):
```

Comment:
Create wod for user is not creating the workout it's only saving an empty workout to the DB